test/testSendNotification.js

```diff
@@ -147,14 +147,12 @@ suite('sendNotification', function() {
 
       assert.equal(requestDetails.headers['content-encoding'], 'aesgcm', 'Check Content-Encoding header');
 
-      ece.saveKey('webpushKey', userCurve, 'P-256');
-
       const decrypted = ece.decrypt(requestBody, {
-        keyid: 'webpushKey',
+        version: 'aesgcm',
+        privateKey: userCurve,
         dh: appServerPublicKey,
         salt: salt,
-        authSecret: urlBase64.encode(userAuth),
-        padSize: 2
+        authSecret: urlBase64.encode(userAuth)
```

Comment:
And we should add tests with aes128gcm here too (but testSelenium is the main one where we need it).